services/worker/src/worker/job_runners/config/parquet_and_info.py

```diff
@@ -671,10 +817,7 @@ def convert_to_parquet(builder: DatasetBuilder) -> List[CommitOperationAdd]:
     builder.download_and_prepare(
         file_format="parquet"
     )  # the parquet files are stored in the cache dir and it fills the info
```

Comment:
does it mean that anyway: the full dataset must be downloaded by the datasets library? It might break things

Reply:
or maybe I didn't understand well: at this point, are the files already downloaded locally?

If we stream the dataset later, it wouldn't make sense to download them, so... I imagine not.

Reply:
this code is `convert_to_parquet` which is for datasets <5GB